Documentation/GitHub/Index.rst

```diff
@@ -9,6 +9,11 @@ How to work with GitHub
 =======================
 
 
+.. tip:: **Looking for help on 'How to edit'?
```

Comment:
I get a rendering error, I think the finishing markup for bold (**) is missing. Will push another commit.